light-zk.js/src/wallet/user.ts

```diff
@@ -60,7 +61,10 @@ import { Idl } from "@coral-xyz/anchor";
 const message = new TextEncoder().encode(SIGN_MESSAGE);
 
 // TODO: Utxos should be assigned to a merkle tree
-// TODO: add support for wallet adapter (no access to payer keypair)
+export enum ConfirmOptions {
```

Comment:
Should we move this to a separate file? /config or constants file

Reply:
in general yes, but I also don't like a lot of files with 4 lines